phpunit.xml

```diff
@@ -1,6 +1,6 @@
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
```

Comment:
why that? I'd stick to the relative path as it's always reflecting the currently installed phpunit version (9.5 on `latest`)

Reply:
The path is different in different versions of PHPUnit, so that doesn't work.

Reply:
I think it [was a bug](https://github.com/sebastianbergmann/phpunit/issues/4405) (or unintentional BC break), that was fixed in `9.3.4`. So you should be safe to use the relative path for the time being and require phpunit `^9.3.4`